frontend/amp/components/elements/TextBlockComponent.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { css } from 'react-emotion';
+import { palette } from '@guardian/pasteup/palette';
+import { pillarPalette } from '../../../pillars';
+
+// tslint:disable:react-no-dangerous-html
+const style = (pillar: Pillar) => css`
+    p {
+        margin: 0 0 12px;
+        padding: 0;
+        font-size: 16px;
+        font-weight: 300;
+        word-wrap: break-word;
+        color: ${palette.neutral[7]};
+    }
+    a {
+        color: ${pillarPalette[pillar].main};
+        text-decoration: none;
+        border-bottom: 1px solid #dcdcdc;
+
+        :hover {
+            border-bottom: 1px solid ${pillarPalette[pillar].main};
+        }
+    }
+`;
+export const TextBlockComponent: React.SFC<{
+    html: string;
+    pillar: Pillar;
+}> = ({ html, pillar }) => (
+    <span
```

Comment:
Can we use React fragments here to avoid the redundant span?

Reply:
Alas, not possible with `dangerouslySetInnerHTML` 😢 

Also I'm torn as to whether it is redundant. A `TextBlockElement` contains multiple `<p>`s so it's kind of semantic to have a tag wrap the component.